src/shared/lib/redirectHelpers.ts

```diff
@@ -66,6 +67,27 @@ export function handleCaseDO(){
 
 }
 
+/*
+ * Handle LinkOut of style /ln?q=TP53:MUT and ln?cancer_study_id=gbm_tcga&q=EGFR+NF1.
+ */
+export function handleLinkOut() {
+    const routingStore:ExtendedRouterStore = getBrowserWindow().routingStore;
+    const currentQuery = routingStore.location.query;
+
+    const data = {
+        case_set_id: 'all',
+        gene_list: currentQuery.q.replace(":MUT","").replace("+",","),
+        cancer_study_list:
+            currentQuery.cancer_study_id ||
+            // use same set of studies as quick search gene query if no
+            // specific study is supplied
+            AppConfig.serverConfig.skin_quick_search_gene_query_session_id ||
+            AppConfig.serverConfig.skin_quick_search_gene_query_cancer_study_list
+    };
+
+    (getBrowserWindow().routingStore as ExtendedRouterStore).updateRoute(data, "/results/mutations");
```

Comment:
we should probably use clear=true and replace=true as last two params in this.   this will prevent other params from getting persisted and also prevent the original ln from getting added to history.